src/ansys/tools/path/path.py

```diff
@@ -38,117 +58,65 @@
+            "Will be unable to cache product executable locations"
         )
 
 CONFIG_FILE = os.path.join(SETTINGS_DIR, CONFIG_FILE_NAME)
 
 
-def _version_from_path(path):
-    """Extract ansys version from a path.  Generally, the version of
-    ANSYS is contained in the path:
-
-    C:/Program Files/ANSYS Inc/v202/ansys/bin/winx64/ANSYS202.exe
-
-    /usr/ansys_inc/v211/ansys/bin/mapdl
-
-    Note that if the MAPDL executable, you have to rely on the version
-    in the path.
-
-    Parameters
-    ----------
-    path : str
-        Path to the MAPDL executable
+def _get_installed_windows_versions(supported_versions=SUPPORTED_ANSYS_VERSIONS):
+    """Get the AWP_ROOT environment variable values for supported versions."""
 
-    Returns
-    -------
-    int
-        Integer version number (e.g. 211).
+    # The student version overwrites the AWP_ROOT env var
+    # (if it is installed later)
+    # However the priority should be given to the non-student version.
+    awp_roots = []
+    awp_roots_student = []
 
-    """
-    # expect v<ver>/ansys
-    # replace \\ with / to account for possible windows path
-    matches = re.findall(r"v(\d\d\d).ansys", path.replace("\\", "/"), re.IGNORECASE)
-    if not matches:
-        raise RuntimeError(f"Unable to extract Ansys version from {path}")
-    return int(matches[-1])
+    for ver in supported_versions:
+        path_ = os.environ.get(f"AWP_ROOT{ver}", "")
+        path_non_student = path_.replace("\\ANSYS Student", "")
 
+        if "student" in path_.lower() and os.path.exists(path_non_student):
+            # Check if also exist a non-student version
+            awp_roots.append([ver, path_non_student])
+            awp_roots_student.insert(0, [-1 * ver, path_])
 
-def _get_available_base_ansys(supported_versions=SUPPORTED_ANSYS_VERSIONS):
-    """Return a dictionary of available Ansys versions with their base paths.
-
-    Returns
-    -------
-    dict[int: str]
-        Return all installed Ansys paths in Windows.
-
-    Notes
-    -----
-
-    On Windows, It uses the environment variable ``AWP_ROOTXXX``.
-
-    The student versions are returned at the end of the dict and with
-    negative value for the version.
-
-    Examples
-    --------
+        else:
+            awp_roots.append([ver, path_])
 
-    >>> from ansys.tools.path.path import _get_available_base_ansys
-    >>> _get_available_base_ansys()
-    {222: 'C:\\Program Files\\ANSYS Inc\\v222',
-     212: 'C:\\Program Files\\ANSYS Inc\\v212',
-     -222: 'C:\\Program Files\\ANSYS Inc\\ANSYS Student\\v222'}
+    awp_roots.extend(awp_roots_student)
+    installed_versions = {ver: path for ver, path in awp_roots if path and os.path.isdir(path)}
+    if installed_versions:
+        LOG.debug(f"Found the following unified Ansys installation versions: {installed_versions}")
+    else:
+        LOG.debug(
+            "No unified Ansys installations found using 'AWP_ROOT' environments. Let's suppose a base path."
```

Comment:
```suggestion
            "No unified Ansys installations found using 'AWP_ROOT' environments."
```

Because you are returning a empty dict (``installed_versions``)